<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registro de ventas</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../iconos/m.ico</normaloff>../iconos/m.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(199, 199, 199);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="detalles"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnDetalle">
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableView" name="tableView">
      <property name="font">
       <font>
        <family>ITC Avant Garde Std Bk Cn</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(199, 199, 199);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QTableView" name="tableView_2"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
